convert_checkpoints.py

```diff
@@ -447,10 +448,10 @@ def convert_hf_gemma_weights(
       state_dict[new_key.replace("qkv_proj", "wk")] = k
       state_dict[new_key.replace("qkv_proj", "wv")] = v
       continue
-    if "o_proj" in key:
-      new_key = new_key.replace("o_proj", "wo")
+
     if new_key != key:
       state_dict[new_key] = state_dict.pop(key)
+  output_ckpt_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
This shouldn't be needed. Output folder will be created in `_export_to_local` https://github.com/google/jetstream-pytorch/blob/811d718c1f93e5ce37182e2c1ec54d3dc0b4aed7/convert_checkpoints.py#L355